### PR TITLE
Add a section on setting up the OBS Virtual Camera

### PR DESCRIPTION
**Pull request by sheepman4267**
## Description

This adds instructions for installing v4l2loopback to enable the OBS Virtual Camera.

### Submitter Checklist

- [x ] Updated the "lastmod" portion at the top of any modified Markdown files.
- [ ] Squashed commits with `git rebase -i` (if needed) - only made one commit

### CODE REVIEW

File: software/open-broadcaster-software/en.md [hunk 1]
```diff
@@ -13,6 +13,28 @@ To install OBS Studio, either open up the Software Center and type `obs-studio`
...
+This should return a version which ends in `.current` or `.lts`. If you are running the "current" kernel, install `v4l2loopback-current`:
+
+```bash
+sudo eopkg install v4l2loopback-current
+```
+or, if you are running the "lts" kernel, install `v4l2loopback`:
+
+```bash
+sudo eopkg install v4l2loopback
+```
```
Comment (JoshStrobl) [inline software/open-broadcaster-software/en.md:34]: Can be condensed a bit.

```
This should return a version which ends in `.current` or `.lts`.

- If you are running the "current" kernel, install `v4l2loopback-current`: `sudo eopkg install v4l2loopback-current`
- If you are running the "lts" kernel, install `v4l2loopback`: `sudo eopkg install v4l2loopback`
```

File: software/open-broadcaster-software/en.md [hunk 2]
```diff
@@ -13,6 +13,28 @@ To install OBS Studio, either open up the Software Center and type `obs-studio`
 ``` bash
 sudo eopkg install obs-studio
 ```
+## Virtual Camera
+
+OBS includes a virtual webcam device which can be used to send your OBS stream to video conferencing programs. In order to use this on Solus, you must install `v4l2loopback`. There are two versions of `v4l2loopback` available in the repository, and you must choose the version that corresponds to the kernel you have installed.
```
Comment (JoshStrobl) [inline software/open-broadcaster-software/en.md:18]: Let's change the following:

```
In order to use this on Solus, you must install `v4l2loopback`. There are two versions of `v4l2loopback` available in the repository, and you must choose the version that corresponds to the kernel you have installed.
```

To...

```
In order to use this on Solus, you must first install from our repository a separate kernel module that is responsible for creating the necessary "loopback" device. There are two versions of this software available in the repository, you must choose the version that corresponds to the kernel you have installed.
```

The reason for this is some folks will just immediately install the first package name they see and not read any further, so we should just avoid using it until we provide the respective instructions.

**Review by JoshStrobl — CHANGES_REQUESTED** [review]

**Comment by JoshStrobl** [comment]
@sheepman4267 Any plans to address this?

**Comment by sheepman4267** [comment]
@JoshStrobl Edits complete, thank you for the feedback!

**Comment by JoshStrobl** [comment]
LGTM, thanks!

**Pull request merged**